iphone/Maps/Core/DeepLink/DeepLinkHandler.swift

```diff
@@ -111,3 +121,9 @@
     }
   }
 }
+
+private extension URL {
```

Comment:
What's the benefit of using extension if the "file" check is used only in one place? Would it be cleaner to make this check at the place of use instead?

Reply:
return scheme == "file" which is used before isn't clear what we are actually checking. Because the main purpose is to
This is my mistake because the URL already has the isFileURL check so this custom check can be safely removed :)
A Boolean that is true if the scheme is file:.
https://developer.apple.com/documentation/foundation/nsurl/1408782-isfileurl